src/EPR.RegulatorService.Frontend.Core/MockedData/Filters/AbstractSubmissionFilters.cs

```diff
@@ -56,4 +56,24 @@ public static class AbstractSubmissionFilters
 
         return query;
     }
+
+    public static IQueryable<AbstractSubmission> FilterBySubmissionYears(this IQueryable<AbstractSubmission> query, int[] submissionYears)
+    {
```

Comment:
If you want to improve the readability of this code you can do

`if (submissionYears?.Any() == true) {`

Also you can use substring operators on the query `x.SubmissionPeriod[^4..]` instead of substring as it's a bit tidier, it will give you the last 4 letters, https://blog.ndepend.com/c-index-and-range-operators-explained/